ui/src/views/AutogenView.vue

```diff
@@ -1256,6 +1256,9 @@ export default {
       }
       var paramName = param.name
       var extractedParamName = paramName.replace('ids', '').replace('id', '').toLowerCase()
+      if (extractedParamName.endsWith('ory')) {
+        extractedParamName = extractedParamName.slice(0, -3) + 'orie'
```

Comment:
would this need to be 'ories' ?

Reply:
@Pearl1594 in that code later at ln 1266 we seem to be using the following, therefore I didn't use ories

```
const possibleName = 'list' + extractedParamName + 's'
```